core/src/test/java/org/apache/iceberg/MetadataTableScanTestBase.java

```diff
@@ -81,18 +81,20 @@ protected void validateTaskScanResiduals(TableScan scan, boolean ignoreResiduals
   }
 
   protected void validateSingleFieldPartition(
-      CloseableIterable<ContentFile<?>> files, int partitionValue) {
+      CloseableIterable<ManifestEntry<? extends ContentFile<?>>> files, int partitionValue) {
```

Comment:
Nit: do you think we can remove "extends ContentFile<?>" here as well?  Compiler seems to be able to figure it out without it.